pkg/entrypoint/metricsbridge/metricsbridge.go

```diff
@@ -268,15 +273,18 @@ func (c *MetricsConfig) runOnce(ctx context.Context) error {
 			if err != nil {
 				return err
 			}
-			if _, err = c.conn.Write(b); err != nil {
+			bytesSent, err := c.conn.Write(b)
+			if err != nil {
 				return err
 			}
-
+			bytesCount += bytesSent
 			metricsCount++
 		}
 	}
+	metricsBridgeMetricsTransferredCounter.Add(float64(metricsCount))
+	metricsBridgeBytesTransferredCounter.Add(float64(bytesCount))
```

Comment:
Why wait until we've broken out of the loop to update this? Should we not do it live?

Reply:
It just felt cleaner to me.

Reply:
My concern here is that the metrics won't be accurate if we end up failing out early due to an error---the bytes count won't be updated even if some bytes have been written already. What do you think?

Reply:
Yes I agree. I'll move those lines into the loop :)